ocs-grpc-api/src/main/proto/ocs.proto

```diff
@@ -102,6 +102,7 @@ message PsInformation {
   string sgsnMccMnc = 2;
   string imsiMccMnc = 3;
   bytes userLocationInfo = 4;
+  string pdpAddress = 5;
```

Comment:
What is this? It is encoded as a string, but is it really something else? An IP number or something, if so I don't think a string is an appropriate carrier in Protobuf (octet stream would be better), but if it's a FQDN then of course text is acceptable (this may become clearer to me as I read through the PR).

Reply:
IPv4 address